.travis.yml

```diff
@@ -25,3 +25,16 @@ after_success:
     pip install codecov;
     codecov;
     fi
+
+deploy:
+  provider: pypi
+  user: betatim
+  # this line together with `setup.cfg` creates universal wheels as long as
+  # skopt is a python only module (no compiled code)
+  distributions: sdist bdist_wheel
+  on:
+    tags: true
```

Comment:
I think so. It will definitely deploy when you create a new release (which is a tag?!) on github.